openstates/wy/bills.py

```diff
@@ -165,7 +160,7 @@ def scrape_digest(self, bill):
                     actor = 'upper'
 
                 date = datetime.datetime.strptime(date, '%m/%d/%Y')
-                bill.add_action(actor, action, date,
+                bill.add_action(actor, action.strip(), date,
```

Comment:
Agreed; it'd be nice, but I wasn't sure if there was a {comm abbrev: comm name} mapping that we could use, and there aren't enough bills yet in the new system to build one from scratch.